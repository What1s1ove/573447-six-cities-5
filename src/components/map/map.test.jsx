import * as React from 'react';
import {render} from 'enzyme';
import Map from './map';

describe(`Map component`, () => {
  it(`should render Map component`, () => {
    const handleMapRender = () => <p>Map</p>;
    const component = render(<Map renderMap={handleMapRender} />);

    expect(component).toMatchSnapshot();
  });
});
