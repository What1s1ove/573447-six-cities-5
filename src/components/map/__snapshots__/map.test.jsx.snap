// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component should render Map component 1`] = `
<p>
  Map
</p>
`;
